resources/views/top.blade.php

```diff
@@ -0,0 +1,104 @@
+
+    <!DOCTYPE html>
+    <html>
+        <head>
+            <meta charset="UTF-8">
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+
+            <title>トップページ</title>
+
+            <!-- Latest compiled and minified CSS -->
+            <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha384-HSMxcRTRxnN+Bdg0JdbxYKrThecOKuH5zCYotlSAcp1+c8xmyTe9GYg1l9a69psu" crossorigin="anonymous">
+
+            <!-- Optional theme -->
+            <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap-theme.min.css" integrity="sha384-6pzBo3FDv/PJ8r2KRkGHifhEocL+1X2rVCTTkUfGk7/0pbek5mMa1upzvWbrUbOZ" crossorigin="anonymous">
+
+            <!-- Latest compiled and minified JavaScript -->
+            <script src="https://stackpath.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha384-aJ21OjlMXNL5UyIl/XNwTMqvzeRMZH2w8c5cRVpzpU8Y5bApTppSuUkhZXN0VxHd" crossorigin="anonymous"></script>
+            
+            <link rel="stylesheet" href="css/style.css" />
+        </head>
+        <body>
+            @include("parts.header")
+
+            <hr>
+            
+            <main>
+                <div class="container"> 
+                        <div class="col-xs-6">
+                            <h1>これはトップページです</h1> 
+                            <h2>これはラジオレビューサイトです。いろいろなラジオ番組を見つけて楽しむことができます。</h2>
+                        </div>
+                        <div class="col-xs-6">
+                            <p class="top_button_1">   
+                                <button type="button" class="btn btn-default btn-lg btn-block"><a href="/">番組を探す</a></button>
+                            </p>
+                            <p class="top_button_2"> 
+                                <button type="button" class="btn btn-default btn-lg btn-block"><a href="/">レビュー投稿する</a></button>
+                            </p>
+                        </div>
+                        </div>
+                </div>
+
+                <hr>
+
+                <div class="container">
+                    <h1 class="text-center">カテゴリーで探す</h1>
+                        <div class="row">
+                            <div class="col-xs">
+                                <div class="col-xs-４"><h3>エリア :</h3></div>
```

Comment:
[imo] チャットで質問いただいていた点ですが、こうすると取り急ぎ改行はされなくなりますね

```suggestion
                                <div class="col-xs-2"><h3>エリア :</h3></div>
```